tests/test_tools.py

```diff
@@ -63,18 +63,36 @@ def test_sag_rectangle(self):
         arr = pydicom.read_file(self.SAG_RECTANGLE_PHANTOM_FILE).pixel_array
         shape_detector = hazen_tools.ShapeDetector(arr=arr)
         centre, size, angle = shape_detector.get_shape('rectangle')
+        centre = hazen_tools.round_tuple(centre)
```

Comment:
Why is this round_tuple necessary? I think it introduces more risk than it's worth. If the "true" answer has too much precision, then we can do an assert almost_equal instead.

Reply:
Ah, wasn't familiar with almost_equal. Will work that in instead. Thanks.

Reply:
Fixed with #fc2f215